proto/fairyring/pep/tx.proto

```diff
@@ -114,3 +115,11 @@ message MsgGetPrivateKeyshares {
 
 message MsgGetPrivateKeysharesResponse {}
 
+message MsgRegisterContract {
```

Comment:
Do we need `MsgDeRegisterContract` too ?

Reply:
i don't think that is required. 

@bowenyou your thoughts on this?